### PR TITLE
Rebranding DocumentDB to Azure Cosmos DB

### PR DESCRIPTION
**Pull request by JiayueHu**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

File: 101-documentdb-account-consistencypolicy-create/README.md [hunk 1]
```diff
@@ -7,11 +7,11 @@
     <img src="http://armviz.io/visualizebutton.png"/>
 </a>
 
-This template will create a DocumentDB Database Account with the name provided, the location set to the same location as the resource group that was used, and the Offer Type set to ***Standard***.
+This template will create a Azure Cosmos DB Database Account with the name provided, the location set to the same location as the resource group that was used, and the Offer Type set to ***Standard***.
```
Comment (SnehaGunda) [inline 101-documentdb-account-consistencypolicy-create/README.md]: "an Azure..."

File: 101-documentdb-account-consistencypolicy-create/azuredeploy.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
       ],
       "defaultValue": "Session",
       "metadata": {
-        "description": "The DocumentDB deault consistency level for this account."
```
Comment (SnehaGunda) [inline 101-documentdb-account-consistencypolicy-create/azuredeploy.json]: default is mispelled

File: 101-documentdb-account-consistencypolicy-create/metadata.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
-  "itemDisplayName": "Create DocumentDB Acc with Consistency",
-  "description": "This template will create a DocumentDB Account with the name provided, the location set to the same location as the resource group that was used, and the Offer Type set to Standard. This template also includes the optional consistencyPolicy property. Using this property you can control the Default Consistency Level for the new Database Account.",
-  "summary": "Create a DocumentDB Database Account with a specified Consistency Policy",
+  "itemDisplayName": "Create Azure Cosmos DB Acc with Consistency",
+  "description": "This template will create a Azure Cosmos DB Account with the name provided, the location set to the same location as the resource group that was used, and the Offer Type set to Standard. This template also includes the optional consistencyPolicy property. Using this property you can control the Default Consistency Level for the new Database Account.",
```
Comment (SnehaGunda) [inline 101-documentdb-account-consistencypolicy-create/metadata.json]: can you check the grammar plz shouldn't it be 'an Azure Comsos DB.."

File: 101-documentdb-account-consistencypolicy-create/azuredeploy.json (hunk 2 above)
Comment (JiayueHu) [reply 101-documentdb-account-consistencypolicy-create/azuredeploy.json]: done

**Comment by JiayueHu** [comment]
Done. thanks a lot for the checking. Updated them.

**Comment by mimig1** [comment]
#sign-off

**Comment by mimig1** [comment]
@JiayueHu - The red x above indicates there was a build error. If you click the Details link, the output says "Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received". @bmoore-msft - I see you've merged most of the most recent PRs. Could you take a look at this one and let us know what looks wrong? Only descriptions and readme's were updated. Also, once the build is successful, do we ping you to get the PR merged? Thanks.

**Comment by mimig1** [comment]
@bmoore-msft - Can this be merged or is there a build error?

**Pull request merged**